watson-developer-cloud/discovery-components
# fix: display DocumentPreview properly for documents with tables

#### What do these changes do/fix?

- Show original document if highlighting table (don't need mappings)
- Add `data-child-begin` and `data-child-end` to table `article` tags
- Choose begin / end nodes as deepest part of DOM that contains offsets
- Update story to test file types with different highlight types

#### How do you test/verify these changes?
Run storybook and / or connect to tooling and verify that documents with tables are able to display in the DocumentPreview.

#### Have you documented your changes (if necessary)?

#### Are there any breaking changes included in this pull request?
No, I think all changes are fixes without changing other significant behavior.
